### PR TITLE
Monkies now take twice as long to starve

### PR DESCRIPTION
**Pull request by Contrabang**
## What Does This PR Do

Monkies now have half the hunger rate as a normal humanoid mob.

## Why It's Good For The Game
Monkies are much smaller, and take 2x effective damage from brute and burn. Allowing the NPC monkies like pun pun and the wizard den monkey to not starve every round would be nice. (They can't eat for themselves anyways, I feel bad for them)

## Testing

I checked that the variable was changed. I'm not gonna wait 3.5 hours to watch a monkey starve

## Changelog
:cl:
tweak: Monkeys and other lesser forms now take twice as long to starve
/:cl:

### CODE REVIEW

**Comment by ArcticValor** [comment]
are you telling me people don't feed pun pun in their free time?

**Comment by DGamerL** [comment]
> are you telling me people don't feed pun pun in their free time?

You SEVERELY overestimate bar/assistant players

**Comment by Gatchapod** [comment]
I question if this is such a great idea in the context of xenomorphs. This basically ensures they will always (unless virologist didn't leave the lab permanently yet) get free hosts, no?
At 80+ pop, this is 4 larvae on the event roll and another 3-ish larvae likely before the announcement pops. It's pretty significant hive against crew that is only beginning to arm up.

But outside of that, I like the idea. Random dead rotting monkeys around the station always bothered me.

File: code/modules/mob/living/carbon/human/species/monkey.dm [hunk 1]
```diff
@@ -38,6 +38,7 @@
 	total_health = 75
 	brute_mod = 1.5
 	burn_mod = 1.5
+	hunger_drain = HUNGER_FACTOR / 2 // twice has slow as normal
```
Comment (lewcc) [inline code/modules/mob/living/carbon/human/species/monkey.dm:41]:
```suggestion
	hunger_drain = HUNGER_FACTOR / 2 // twice as slow as normal
```

**Review by lewcc — COMMENTED** [review]
the nittiest of nitpicks

File: code/modules/mob/living/carbon/human/species/monkey.dm (hunk 1 above)
Comment (Contrabang) [reply code/modules/mob/living/carbon/human/species/monkey.dm:41]: i meant to put `as`, I swear
minor-spelling-mistake

**Review by S34NW — APPROVED** [review]

**Pull request merged**